src/utils/hooks/send-notif-to-related-users.js

```diff
@@ -0,0 +1,42 @@
+const sendMessage = require('../send-bot-notification');
+
+const User = require('../../services/user/user-model');
+
+const DEPARTMENT_HEAD_ROLE = require('../../constants/roles').DEPARTMENT_HEAD;
+
+// Assume that a before hook attach logInfo in proper format already
+const sendNotifToRelatedUsers = () => (hook) => {
+  // Find all related users' bot ids.
+  let relatedUsers = [];
+  const relatedDepartment = hook.data.logInfo.notifyDepartments;
+  const findUserPromises = relatedDepartment.map((x) => User.find({ department: x, role: DEPARTMENT_HEAD_ROLE }));
+  Promise.all(findUserPromises)
+    .then(results => {
+      // TODO(A): Using correct facebookId (now using fb login's id instead of bot's id.)
+      for (let i = 0; i < results.length; ++i) {
+        const fbIdList = results[i].map((x) => x.facebookId);
```

Comment:
Anyway, due to we are approaching a deadline and this might not be critical, you can send a new PR to fix this issue.